ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1020,6 +1027,7 @@ def grad_fn(all_args):
         )
         grads_np_from_gt_flat = flatten_and_to_np(ret=grads_from_gt)
         ivy.previous_backend()
+        ivy.unset_backend()
```

Comment:
I think this is old changes from old commit, please also remove this (repetition with the existing code - name change)